### ISSUE TITLE
How to access scripts methods from another script inside nunu

### ISSUE THREAD

**Issue by jfa257** [tentone/nunuStudio#248]
Hi,

 I have a question: lets say I have a script called home, an object script in the scene. Now from another object script also in the scene (or not as well?) called let's say visitor I want to call home.Ringbell() function so I can start visiting (just an example) How to achieve this on script?

 Thanks in advance!

**Comment by tentone** [comment]
One way to achieve this is to use the self pointer to attach the method that you want to access externally to the script object.

For example

```
function ringBell(){....)
self.ringBell = ringBell;
```

On the other script get the object by name and call it as.

`object.ringBell()`

**Comment by jfa257** [comment]
Oh great! nice tip. Thanks!

**Comment by jfa257** [comment]
Hi,

 my first try didn't worked. Truth be told, I attempted expose not methods this time around, but booleans and array variables. I tried:

var myarr = [];
var mybool = false;

function initialize()
{
  self.myarr = myarr ///after pushing items inside myarr///
  self.mybool = mybool;
}

 and then on another script:

 var myScr = scene.getObjectByName("nameOfContainerScriptObjectOnScene");
 
 function usage()
{
 myScr.myarr ....
 myScr.mybool ....
}

 what I did wrong?

thanks in advance!!

Jorge
